### DISCUSSION TITLE
Disable default site for source

### DISCUSSION THREAD

**Discussion by Azmodeszer** [bb-Ricardo/netbox-sync#337] [Q&A] [answered]
Is it possible to prevent the sync from automatically assigning the site "[Source]: [Name]" to all objects imported from a specific source? We have VM hosts from more than one site assigned to the same cluster, so previously the solution was to just leave the site field empty for the cluster (otherwise Netbox would complain about devices from another site being assigned to it). I tried setting all the `cluster_site_relations` to `<NONE>`, but that doesn't prevent the default virtual site from being assigned to it. 

I assume this is very much intentional, since the site is a required field for devices. However, my plan is to set host sites via the `host_site_relation` option individually anyway.

Thanks.

**Comment by bb-Ricardo** [comment, marked as answer]
Hi,

There is actually a current PR #333 which I haven't had time to merge.

If you like, you can check out the code and try if this works for you. Would be great to help testing this feature.

**Reply by Azmodeszer** [reply]
lol, I'm an idiot. Had the answer staring me in the face this whole time. Yeah, I had actually implemented that PR already and it works, all I had to do was assign the sites to the actual hosts with `host_site_relation`. Thanks
